compiler/optimizer/UseDefInfo.hpp

```diff
@@ -264,15 +264,15 @@ class TR_UseDefInfo
    bool    isExpandedUseDefIndex(uint32_t index)
                { return index >= _numExpandedDefOnlyNodes && index < getNumExpandedDefNodes(); }
 
-   bool getDefsForSymbol(BitVector &defs, int32_t symIndex, AuxiliaryData &aux)
+   bool getDefsForSymbol(TR_BitVector &defs, int32_t symIndex, AuxiliaryData &aux)
       {
-      defs.Or(aux._defsForSymbol[symIndex]);
-      return !defs.IsZero();
+      defs |= (*aux._defsForSymbol[symIndex]);
```

Comment:
the * binding here is a bit odd - I think it is derefing the result of the [], but can the parens make this clear ie *() or (*()) rather than (*...)